src/express/outgoing/controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Request, Response } from 'express';
+import mongoose from 'mongoose';
+import * as OutgoingManager from './manager';
+
+export const createOutgoingFile = async (req: Request<{}, any, {}, any>, res: Response) => {
+    res.json(await OutgoingManager.createOutgoingFile(req.user.id, req, req.query));
+};
+
+export const getOutgoingFile = async (
+    req: Request<{ userId: string; fileId: mongoose.Types.ObjectId }>,
+    res: Response,
+) => {
+    res.json(await OutgoingManager.getOutgoingFileById(req.params.userId, req.params.fileId));
```

Comment:
we get user from req.user.id not from params in all routes
```suggestion
    res.json(await OutgoingManager.getOutgoingFileById(req.user.id, req.params.fileId));
```

Reply:
done